packages/avatars/src/components/Avatar.tsx

```diff
@@ -5,27 +5,27 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { Children } from 'react';
-import PropTypes from 'prop-types';
+import React, { Children, HTMLAttributes } from 'react';
 import { StyledAvatar, StyledText } from '../styled';
 
-const SIZE = {
-  EXTRASMALL: 'extrasmall',
-  SMALL: 'small',
-  MEDIUM: 'medium',
-  LARGE: 'large'
-};
-
-const STATUS = {
-  AVAILABLE: 'available',
-  ACTIVE: 'active',
-  AWAY: 'away'
-};
+interface IAvatarProps extends HTMLAttributes<HTMLElement> {
+  /** Set the avatar background color */
+  backgroundColor?: string;
+  /** Set the color for child SVG or `Avatar.Text` components */
+  foregroundColor?: string;
+  /** Set the color of the surface behind the avatar – used to manipulate the inner status rings */
+  surfaceColor?: string;
+  /** Applies system styling */
+  isSystem?: boolean;
+  size?: 'extrasmall' | 'small' | 'medium' | 'large';
+  status?: 'available' | 'away';
```

Comment:
I noticed on [line 196](https://github.com/zendeskgarden/react-components/pull/477/files#diff-a6953e6648bf616b20ee1d05b13b0c76R196) in the `IStyledAvatarProps` that `status` includes an optional `active` status, which isn't included in the `status` for `IAvatarProps`. 

Should `active` be added here too?

```ts
status?: 'available' | 'active' | 'away';
```

Reply:
Good question, but no. The styling is directly impacted by the 3-part status. In the higher-abstraction `Avatar` component, we only deal with the two statuses – and any occurrence of `badge` overrides to ensure that the component is styled as `active`.

Reply:
Thanks for the explanation!